spec.html

```diff
@@ -40856,9 +40857,9 @@ <h1>Module Namespace Objects</h1>
     <p>A Module Namespace Object is a module namespace exotic object that provides runtime property-based access to a module's exported bindings. There is no constructor function for Module Namespace Objects. Instead, such an object is created for each module that is imported by an |ImportDeclaration| that includes a |NameSpaceImport|.</p>
     <p>In addition to the properties specified in <emu-xref href="#sec-module-namespace-exotic-objects"></emu-xref> each Module Namespace Object has the following own property:</p>
 
-    <emu-clause id="sec-@@tostringtag">
-      <h1>@@toStringTag</h1>
-      <p>The initial value of the @@toStringTag property is the String value *"Module"*.</p>
+    <emu-clause oldids="sec-@@tostringtag" id="sec-%symbol.tostringtag%">
```

Comment:
This should have a more specific ID that is unlikely to conflict:

```suggestion
    <emu-clause oldids="sec-@@tostringtag" id="sec-module-namespace-objects-tostringtag">
```
or, depending on which section ID style gets chosen:
```suggestion
    <emu-clause oldids="sec-@@tostringtag" id="sec-module-namespace-objects-%symbol.tostringtag%">
```

Reply:
it won’t conflict with anything else because there will never be another top level section for this value

Reply:
Well, I still feel like we should fix the section ID.

Reply:
I don't see the value of that; `sec-%symbol.tostringtag%` isn't going to ever conflict with anything else. this change just makes the URL longer for no value.